Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG CKAN_VERSION=2.9
-FROM openknowledge/ckan-dev:${CKAN_VERSION}
+FROM openknowledge/ckan-dev:${CKAN_VERSION}.7
```

Comment:
is this supposed to be there?

Reply:
That is the temporary fix.  This extension does not work with anything newer right now...